# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hydrogen/string.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
default option is to build qt without STL support...
- Sebastian Czech <t2_@arcor.de>

--- hydrogen-0.9.1/hydrogen_gui/PreferencesDialog.cpp.orig	2004-11-28 16:09:19.000000000 +0100
+++ hydrogen-0.9.1/hydrogen_gui/PreferencesDialog.cpp	2005-03-27 13:42:41.000000000 +0200
@@ -84,7 +84,7 @@
 
 	// ALSA
 	m_pAlsaBufferSizeTxt->setValue( pPreferencesMng->getAlsaBufferSize() );
-	m_pAlsaDeviceEdit->setText( pPreferencesMng->getAlsaAudioDevice() );
+	m_pAlsaDeviceEdit->setText( pPreferencesMng->getAlsaAudioDevice().c_str() );
 	switch ( pPreferencesMng->getAlsaAudioDeviceSampleRate() ) {
 		case 44100:
 			m_pAlsaSampleRateComboBox->setCurrentItem( 0 );
